app/src/main/java/com/goryachok/forecastapp/api/OpenWeatherMapAPI.kt

```diff
@@ -0,0 +1,52 @@
+package com.goryachok.forecastapp.api
+
+
+import com.goryachok.forecastapp.base.RemoteEntity
+import kotlinx.coroutines.Deferred
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+
+interface OpenWeatherMapAPI {
+
+    companion object {
+        const val API_KEY = "f19f55d718abb04df88d9190337bd5b9"
+        const val DOMAIN = "https://api.openweathermap.org/"
+    }
+
+//    @GET("data/2.5/weather")
+//    suspend fun getWeatherByCityAsync(
```

Comment:
Please either add "TODO" if you're planning to use commented code, or remove unused commented code.

Reply:
Removed